.github/workflows/homebrew-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Homebrew Tap
+on:
+  release:
+    types: [released]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - id: version
+      name: Compute version number
+      run: |
+        echo "::set-output name=result::$(echo $GITHUB_REF | sed -e "s/^refs\/tags\/v//")"
+    - id: hash
+      name: Compute release asset hash
+      uses: mjcheetham/asset-hash@v1
+      with:
+        asset: Installers_macOS_Release.zip
+        hash: sha256
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This token is automatically created. It only has public read access to GitHub repos, which is all we need to compute the hash from a released Scalar asset.